### PR TITLE
Export active viewport as image

### PR DESCRIPTION
**Pull request by linasburneika**
Export active viewport as png image

### CODE REVIEW

File: src/sampleManifest.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ import { getUnifiedSelectionTreeSpec } from "./frontend-samples/tree-samples/uni
 import { getPresentationTreeSpec } from "./frontend-samples/tree-samples/presentation-tree/sampleSpec";
 import { getToolbarButtonSample } from "./frontend-samples/app-ui-samples/toolbar-button-provider-sample/sampleSpec";
 import { getViewportFrontstageSample } from "./frontend-samples/app-ui-samples/viewport-frontstage-sample/sampleSpec";
+import { getImageExportSpec } from "frontend-samples/image-export/sampleSpec";
```
Comment (williamkbentley) [inline src/sampleManifest.tsx:47]: Can you sort these imports?

**Comment by williamkbentley** [comment]
I'm not really loving the thumbnail. It gets the point across, but it is currently specific to Chrome. Maybe you could request Jon for some feedback, or to create a better thumbnail?

**Comment by linasburneika** [comment]
> 
> 
> I'm not really loving the thumbnail. It gets the point across, but it is currently specific to Chrome. Maybe you could request Jon for some feedback, or to create a better thumbnail?

Redesigned the thumbnail to be more generic, though it's hard to illustate this "feature"

**Review by williamkbentley — COMMENTED** [review]
LGTM. I like the new thumbnail, but Jon or Josh may have thoughts on it

**Review by Josh-Schifter — APPROVED** [review]

**Pull request merged**